tests/test_water_mask.py

```diff
@@ -16,3 +17,32 @@ def test_create_water_mask(tmp_path, test_data_dir):
     assert info['bands'][0]['type'] == 'Byte'
     assert info['bands'][0]['minimum'] == 1
```

Comment:
should it be assert info['bands'][0]['minimum'] == 0 ?

Reply:
This particular geotiff is inland, so I'm asserting that all the pixels have a value of 1.

Reply:
OK. I got it. forget about it.